### ISSUE TITLE
Suggestion: option to not exit hint mode until ESC

### ISSUE THREAD

**Issue by ycmjason** [dexterleng/vimac#147]
Thanks for the amazing app.

I have a suggestion to add an option to allow "persistent hint mode". Currently, it goes back to "normal mode" after selecting a hint. But I find more operations I want to do require multiple clicks. E.g. going to the setting page of Chrome requires 2 clicks.

**Comment by ashbaker1** [comment]
Agree, would be useful (if this mode can be assigned to a separate shortcut).

**Comment by ultraGentle** [comment]
Maybe persist while caps-lock is on?
